src/commands/install.ts

```diff
@@ -16,6 +24,10 @@ export default class Install extends Command {
 
   static flags = {
     ...Command.flags,
+    prefix: flags.string({
+      char: 'p',
+      description: 'Path prefix indicating where the install command should execute from',
+    }),
```

Comment:
We should use the `services` flag like we do in deploys for this iteration. That allows us to specify multiple install paths gracefully if we want to code-gen for multiple folders.

Reply:
That seems like it has potential to get really messy, plus shouldn't someone be able to use the `-r` flag if they want to do this?

Reply:
That only works for a single service and it's deps. Doesn't work for peer services that don't relate to each other. 

It likely won't be common to use it that way, but it creates a standard convention that matches the other commands.